actionview/lib/action_view/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Engine # :nodoc:
 
     initializer "action_view.per_request_digest_cache" do |app|
       ActiveSupport.on_load(:action_view) do
-        if app.config.consider_all_requests_local
+        if app.config.consider_all_requests_local && !Rails.env.test?
           app.executor.to_run ActionView::Digestor::PerExecutionDigestCacheExpiry
```

Comment:
The point of the config settings is to abstract away the environment definitions. People can define their own environments, for example.

Unless I'm confused, I'm just suggesting we change the `if` to use `! cache_template_loading` where it currently uses `consider_all_requests_local`, so I'm not sure where the overkill is.